tests/e2e/iothub_e2e/aio/conftest.py

```diff
@@ -66,7 +21,7 @@ def event_loop():
 
 
 @pytest.fixture(scope="function")
-async def brand_new_client(device_identity, client_kwargs, service_helper, device_id, module_id):
+async def session_object(device_identity, client_kwargs, service_helper, device_id, module_id):
```

Comment:
nit: why `session_object` instead of just `session`? Reading through the rest of the tests, I find it affects readability a bit to refer to `session_object`. NBD either way, just curious.

Reply:
session is a pytest word. I think there might be a fixture named "session" and we use session scope for a bunch of things.